graphql/client.go

```diff
@@ -53,10 +54,27 @@ type client struct {
 // The typical method of adding authentication headers is to wrap the client's
 // Transport to add those headers.  See example/caller.go for an example.
 func NewClient(endpoint string, httpClient Doer) Client {
+	return newClient(endpoint, httpClient, http.MethodPost)
+}
+
+// NewClientUsingGet returns a Client which makes requests to the given endpoint,
+// suitable for most users.
+//
+// The client makes GET requests to the given GraphQL endpoint using standard
+// GraphQL HTTP-over-JSON transport.  It will use the given http client, or
```

Comment:
I don't think this sentence is accurate for GET -- it's not using JSON in this case.  This is a great place to describe what query params you pass to the server.

Reply:
Good call out, will update